src/ocean/util/log/PeriodicTrace.d

```diff
@@ -187,7 +187,10 @@ struct PeriodicTracer
     {
         va_list ap;
 
-        va_start(ap, __va_argsave);
+        static if (__VERSION__ > 2071)
+            va_start(ap, fmt);
+        else
+            va_start(ap, __va_argsave);
```

Comment:
Well we'll switch to CT vararg in v4.0.0 so I'm not sure this fix is worth it, is it ?

Reply:
Probably. It looks like https://github.com/sociomantic-tsunami/ocean/pull/93 will require some breaking changes to switch to vanilla DMD one way or another, so your point makes perfect sense.

Reply:
FYI: I have kept it so we can enable travis testing immediately. Feel free to kill it when CT vararg takes over.